packages/messaging/README.md

```diff
@@ -34,16 +34,17 @@ npm install {path to node package file}
 ```
 
 **Podfile setup for `@adobe/react-native-aepmessaging` v1.0.0-beta.2 with in-app messaging support**  
-The In app Message APIs depends on the AEP Messaging, v1.1.0. This version is not yet published to the Cocoapods but is available in the public [github repository](https://github.com/adobe/aepsdk-messaging-ios/tree/staging). Add the following pod dependency in your applications Podfile under the application target. 
+The In app Message APIs depends on the AEP Messaging, v1.1.0. This version is not yet published to the Cocoapods but is available in the public [github repository](https://github.com/adobe/aepsdk-messaging-ios/tree/staging). Add the following pod dependency in your applications Podfile under the application target.
```

Comment:
nit: we usually use "In-app" when talking about in-app messages

Reply:
caught by a whitespace change 😄 

This text didn't actually change at all, but you're right, we normally hyphenate `in-app`